cilium-cli/connectivity/builder/client_egress_l7_tls_deny_without_headers.go

```diff
@@ -14,7 +14,7 @@ type clientEgressL7TlsDenyWithoutHeaders struct{}
 func (t clientEgressL7TlsDenyWithoutHeaders) build(ct *check.ConnectivityTest, templates map[string]string) {
 	// Test L7 HTTPS interception using an egress policy on the clients.
 	// Fail to load site due to missing headers.
-	newTest("client-egress-l7-tls-deny-without-headers", ct).
+	newTest("seq-client-egress-l7-tls-deny-without-headers", ct).
```

Comment:
Not sure if I understood properly, but if you want to run the test sequentially you need to move it from:
https://github.com/cilium/cilium/blob/main/cilium-cli/connectivity/builder/builder.go#L256

to:
https://github.com/cilium/cilium/blob/main/cilium-cli/connectivity/builder/builder.go#L289

Reply:
🤦 Thanks!